src/reader.rs

```diff
@@ -19,12 +19,42 @@ enum PipedChar {
     EOF,
 }
 
+#[derive(Clone)]
 pub enum ReadUntil {
     String(String),
     Regex(Regex),
     EOF,
     NBytes(usize),
-    Any(Vec<ReadUntil>),
```

Comment:
nice that you got rid of this, I was always under the impression that the Any doesn't belong into this enum because it is "one level above"